scripts/data_ingestion/download_statcast_pitch_level.py

```
@@ -0,0 +1,145 @@
+    except Exception as e:
+        print(f"  ❌ Error downloading Statcast data: {e}")
+        return False
+
+
+def download_statcast_season(season: int) -> bool:
+    """Download Statcast data for an entire season."""
+    print(f"Downloading Statcast data for {season} season...")
+    
+    # Statcast data available from 2015 onwards
+    if season < 2015:
+        print(f"  ⚠️  Statcast data only available from 2015 onwards")
```

Ruff (0.15.10)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)